app-misc/mosquitto/mosquitto-1.4.15.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+inherit systemd user toolchain-funcs python-any-r1
+
+DESCRIPTION="An Open Source MQTT v3 Broker"
+HOMEPAGE="http://mosquitto.org/"
+SRC_URI="http://mosquitto.org/files/source/${P}.tar.gz"
+LICENSE="EPL-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="bridge examples +persistence +srv ssl tcpd websockets"
+
+RDEPEND="tcpd? ( sys-apps/tcp-wrappers )
+	ssl? ( dev-libs/openssl:0= )"
+DEPEND="${RDEPEND}
+	${PYTHON_DEPS}
+	srv? ( net-dns/c-ares )
+	websockets? ( net-libs/libwebsockets )"
+
+pkg_setup() {
+	enewgroup mosquitto
+	enewuser mosquitto -1 -1 -1 mosquitto
+}
+
+src_prepare() {
+	eapply "${FILESDIR}/${PN}-1.4.10-conditional-tests.patch"
+	if use persistence; then
+		sed -i -e "s:^#autosave_interval:autosave_interval:" \
+			-e "s:^#persistence false$:persistence true:" \
+			-e "s:^#persistence_file:persistence_file:" \
+			-e "s:^#persistence_location$:persistence_location /var/lib/mosquitto/:" \
+			mosquitto.conf || die
+	fi
+
+	# Remove prestripping
+	sed -i -e 's/-s --strip-program=${CROSS_COMPILE}${STRIP}//'\
+		client/Makefile lib/cpp/Makefile src/Makefile lib/Makefile || die
+
+	python_setup
+	python_fix_shebang test
+	eapply_user
+}
+
+src_configure() {
+	LIBDIR=$(get_libdir)
+	makeopts=(
```

Comment:
It's up to you but I'd say it's generally cleaner to add a wrapper function that adds all the args than to keep them in global variable.